Formula/fastq-tools.rb

```diff
@@ -0,0 +1,34 @@
+class FastqTools < Formula
+  desc "Small utilities for working with fastq sequence files"
+  homepage "https://github.com/dcjones/fastq-tools"
+  url "https://github.com/dcjones/fastq-tools/archive/refs/tags/v0.8.3.tar.gz"
```

Comment:
Minor style change to match majority of GitHub formulae:
```suggestion
  url "https://github.com/dcjones/fastq-tools/archive/v0.8.3.tar.gz"
```